packages/expo-haptics/src/Haptics.types.ts

```diff
@@ -45,3 +45,84 @@ export enum ImpactFeedbackStyle {
    */
   Rigid = 'rigid',
 }
+
+export enum AndroidHaptics {
+  /**
+   * A haptic effect to signal the confirmation or successful completion of a user interaction.
+   */
+  Confirm = 'confirm',
+  /**
+   * A haptic effect to signal the rejection or failure of a user interaction.
+   */
+  Reject = 'reject',
+  /**
+   * The user has started a gesture (e. g. on the soft keyboard).
```

Comment:
```suggestion
   * The user has started a gesture (for example, on the soft keyboard).
```

Sorry, I missed this one :)